src/Mod/Part/App/PrimitiveFeature.cpp

```diff
@@ -665,7 +672,7 @@ App::DocumentObjectExecReturn *Cone::execute()
             BRepPrimAPI_MakeCone mkCone(Radius1.getValue(),
                                         Radius2.getValue(),
                                         Height.getValue(),
-                                        Angle.getValue()/180.0f*M_PI);
+                                        angle/180.0f*M_PI);

```

Comment:
nitpick: maybe use `Base::toRadians`, as elsewhere in this file?

Reply:
Absolutely a legitimate point, but M_PI / 180 appears about 68 times, and 180 / M_PI appears 10 times, and 180 * M_PI appears 9 times and 180.0f*M_PI appears 9 times (8 in this very file) ...  you get the point.  I think that should be somebody's separate PR to clean up all the angle/radian conversions, and not part of this extremely narrowly focused piece of nastiness, since there isn't a clear line on where to stop.  Just this one?  This one and the one in the clause above it?  All of them in this file?  All of them in App?

Reply:
I see your point and I agree that this very OCCT workaround should not be mixed with general cleanup.
However, if I touch a line of code anyway, I can "touch it for the better".  -- Your mileage may vary and I accept that.

Reply:
@jbaehr can you make an Issue for that? We can discuss in tomorrow's review meeting, but I'm inclined to merge this as-is and then promote a bigger-picture refactor to make the suggested change.